index.html

```diff
@@ -15,47 +14,232 @@
-         <li><a href="#"><img src="./Assets/social/medium.svg" alt="Medium"></a></li>
-         <li><a href="https://github.com/dvcorreia1984"><img src="./Assets/social/github.svg" alt="Github"></a></li>
-         <li><a href="#"><img src="./Assets/social/angellist.svg" alt="Angellist"></a></li>
-        </ul>
+
+    <div class="griditem">
+      <div class="placeholder"></div>
+        <div class=whitespace>
+          <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
+              <ul class="minimenu">
+                <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
+                <li><div class="button2"><button2>CSS</button2><div><li>  
+                <li><div class="button2"><button2>Javascript</button2><div><li>
+                <li><div class="button2"><button2>HTML</button2><div><li>    
+              </ul>
+              <button class="button">See Project</button>
+          </div>
+        </div>
       </div>
+    </div>
+  </div>
 
 
+  <div class="griditem">
+    <div class="placeholder"></div>
+      <div class=whitespace>
+        <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <ul class="minimenu">
+              <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
+              <li><div class="button2"><button2>CSS</button2><div><li>  
+              <li><div class="button2"><button2>Javascript</button2><div><li>
+              <li><div class="button2"><button2>HTML</button2><div><li>    
+            </ul>
+            <button class="button">See Project</button>
+        </div>
+      </div>
     </div>
+  </div>
+</div>
 
-  </section>
-  
-  </main>
+<div class="griditem">
+  <div class="placeholder"></div>
+    <div class=whitespace>
+      <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
+          <ul class="minimenu">
+            <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
+            <li><div class="button2"><button2>CSS</button2><div><li>  
+            <li><div class="button2"><button2>Javascript</button2><div><li>
+            <li><div class="button2"><button2>HTML</button2><div><li>    
+          </ul>
+          <button class="button">See Project</button>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+</div>
 
-  <footer class="footer">
-  
-  </footer>
+<div class="griditem">
+  <div class="placeholder"></div>
+  <div class="whitespace">
+    <div>
+      <h3>Multi-Post Stories<br> Gain+Glory</h3>
+      <ul class="minimenu">
+        <li><button class="button2">Ruby on Rails</button></li>
+        <li><button class="button2">CSS</button></li>
+        <li><button class="button2">Javascript</button></li>
+        <li><button class="button2">HTML</button></li>
+      </ul>
+      <button class="button">See Project</button>
+    </div>
+  </div>
+</div>
+
+
+<div class="about">
+  <div>
+    <h1 id="h1p3">About me</h1>
```

Comment:
- [ ] As suggested by the previous reviewer,  Kindly change this `<h1>` tag to another `heading tag(like h2,h3 etc.)` tag. It is better for your page to have just one `<h1>` tag, this makes your website semantic and easily readable by browsers. 👍